docs/source/host-collect-analyze/overview.md

```diff
@@ -0,0 +1,65 @@
+---
+title: Overview
+description: Host collectors and analyzers in support bundles
+---
+
+> New in v0.40.0 of Troubleshoot!
+
+## Introduction
+If you need to collect and analyze information that is not available when using in-cluster collectors, you can use host collectors to gather information about the environment, such as CPU, memory, available block devices, and so on. This is especially useful when you need to debug a Kubernetes cluster that is down.
+
+### Differences Between In-Cluster and Host Collectors
+[In-cluster collectors](https://troubleshoot.sh/collect/collectors), specified with the `collectors` property in the `SupportBundle` specification, collect information from a running Kubernetes cluster or schedule a resource in the cluster to dynamically generate data. 
+
+Host collectors gather information directly from the host that they are run on and do not have Kubernetes as a dependency. They can be used to test network connectivity, collect information about the operating system, and gather the output of provided commands.
```

Comment:
This seems related to Chris's feedback. I think it might be even clearer to mention that if you have a multi-node cluster, you need to run host collectors independently on each node if you want to see information about the whole cluster. That's implied, but a call out might make it immediately clear.

Reply:
Yeah, that makes sense